Leyrin-Aguilar/tarea-3/Initialize_my_name/script.js

```diff
@@ -0,0 +1,9 @@
+function initializeNames(name) {
+  var n = name.split(" ");
+  if (n.length < 3) return name;
+  for (i = 1; i < n.length - 1; i++) {
```

Comment:
@LeyrinAguilar  , use modern approaches for array operations like forEach, map, reduce, in fact this one could be a one liner like oddOrEven solution